src/lib.rs

```diff
@@ -18,6 +18,9 @@
 //!
 //! - `upnp-description`: Adds support for accessing the UPnP description of a bridge. See the
 //! [`bridge::Description`] struct for more information.
+//! - `old-api`: Minimal effort support for older api versions. Useful for users of the no longer
+//! supported Hue v1 bridge. This lowers the supported API version to `1.16` not all features 
```

Comment:
Can you remove the trailing space here (otherwise the CI will fail).

Also sorry for the nitpicking

Reply:
I was wondering why it failed, should be done now. Thanks for taking time out of your day to merge this.

Reply:
Clippy still fails but that's unrelated to your changes.